pages/common/gau.md

```diff
@@ -0,0 +1,33 @@
+# gau
+
+> Fetches known URLs from AlienVault's Open Threat Exchange, the Wayback Machine, and Common Crawl of domain(s).
```

Comment:
```suggestion
> Get All URLs: fetch known URLs from AlienVault's Open Threat Exchange, the Wayback Machine, and Common Crawl for any domains.
```
- Maybe we can put the acronym before the description and remove the line below it?
- We always use the imperative mood over the indicative mood
- I think "for" is better than "of" in this context
- We don't generally use "(s)" in the pages, we don't need to be that specific for this aspect of the commands. I've added an any to don't make it weird.